File: 06-pattern_printting.cpp
#include<iostream>
using namespace std;
int main(){
    int row,col;
    // for(row=1; row<=5; row++){
    //     for(col=1; col<=5; col++){
    //         cout<<"*"<<" ";
    //     }
    //     cout<<endl;
    // }

    // for(row=1; row<=5;row++){
    //     for(col=1;col<=5;col++){
    //         cout<<row<<" ";
    //     }
    //     cout<<endl;
    // }




    // 1 4 9 16 25 
    // 1 4 9 16 25 
    // 1 4 9 16 25 
    // 1 4 9 16 25 
    // 1 4 9 16 25
    // for(row=1;row<=5;row=row+1){
    //     for(col=1; col<=5;col++){
    //         cout<<col*col<<" ";
    //     }
    //     cout<<endl;

    // }



    // a a a a a 
    // b b b b b 
    // c c c c c 
    // d d d d d 
    // e e e e e 
    for(row=1;row<=5;row++){
        for(col=1; col<=5; col++){
            char name='a'+(row-1);
            cout<<name<<" ";
        }
        cout<<endl;

    }
}

File: 06-pattern_printting.py
# for i in range(1,6):
#     for j in range(1,6):
#         print("*", end=" ")
#     print()
    
    
# for row in range(1,6):
#     for col in range(1,6):
#         print(col, end=" ")
#     print()


# 1 4 9 16 25 
# 1 4 9 16 25 
# 1 4 9 16 25 
# 1 4 9 16 25 
# 1 4 9 16 25
# for row in range(1,6):
#     for col in range(1,6):
#         print(col*col,end=" ")
#     print()


# a a a a a 
# b b b b b 
# c c c c c 
# d d d d d 
# e e e e e 
for row in range(1,6):
    for col in range(1,6):
        name=chr(ord("a")+(row-1))
        print(name,end=" ")
    print();